CMakeLists.txt

```diff
@@ -246,7 +246,7 @@ add_glob_target(
   FLAGS -x -a
   GLOB_DIRS scripts
   GLOB_PAT *.sh
-  TOUCH_STRATEGY SINGLE)
+  TOUCH_STRATEGY PER_DIR)
```

Comment:
`lintsh` is basically the "ideal" candidate for `SINGLE`, so if we're not using it here it makes me question its efficacy. Maybe we should just remove the `SINGLE` option and just use `PER_DIR` since it roughly does what `SINGLE` does?